libkbfs/folder_block_ops.go

```diff
@@ -60,6 +66,8 @@ func (si *syncInfo) DeepCopy(codec Codec) (*syncInfo, error) {
 			return nil, err
 		}
 	}
+	newSi.toCleanIfUnused = make([]mdToCleanIfUnused, len(si.toCleanIfUnused))
+	copy(newSi.toCleanIfUnused, si.toCleanIfUnused)
```

Comment:
I meant for the bpses inside `toCleanIfUnused`, not the main outer one (which I agree should remain deep-copied).